<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="717" started-at="2017-02-08T07:40:58Z" finished-at="2017-02-08T07:40:59Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="717" started-at="2017-02-08T07:40:58Z" finished-at="2017-02-08T07:40:59Z">
      <class name="com.oneTrust.page.test.OneTrustTest">
        <test-method status="FAIL" signature="contactTest()[pri:0, instance:com.oneTrust.page.test.OneTrustTest@77f03bb1]" name="contactTest" duration-ms="648" started-at="2017-02-08T13:10:58Z" finished-at="2017-02-08T13:10:59Z">
          <exception class="java.lang.NoClassDefFoundError">
            <message>
              <![CDATA[org/apache/xmlbeans/impl/common/SystemCache]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NoClassDefFoundError: org/apache/xmlbeans/impl/common/SystemCache
	at org.apache.poi.POIXMLDocument.init(POIXMLDocument.java:77)
	at org.apache.poi.POIXMLDocument.<init>(POIXMLDocument.java:63)
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:285)
	at com.oneTrust.utility.ExcelUtil.setExcelFile(ExcelUtil.java:32)
	at com.oneTrust.page.test.OneTrustTest.contactTest(OneTrustTest.java:25)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
Caused by: java.lang.ClassNotFoundException: org.apache.xmlbeans.impl.common.SystemCache
	at java.net.URLClassLoader.findClass(Unknown Source)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	... 29 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NoClassDefFoundError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- contactTest -->
      </class> <!-- com.oneTrust.page.test.OneTrustTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
